HLTriggerOffline/Scouting/plugins/ElectronEfficiencyPlotter.h

```diff
@@ -0,0 +1,60 @@
+#ifndef ElectronEfficiencyPlotter_H
```

Comment:
the inclusion guards do not conform to rule 4.1 https://cms-sw.github.io/cms_coding_rules.html#4--technical-coding-rules-1. 
Incidentally, can't the header be merged with the plugin implementation?